npm/src/language-service/language-service.ts

```diff
@@ -97,11 +97,15 @@ export class QSharpLanguageService implements ILanguageService {
     offset: number
   ): Promise<ICompletionList> {
     const code = this.code[documentUri];
+    if (!code) {
```

Comment:
Due to "" being 'falsy', this will be false here if the document is an empty string.